docs/source/ReferenceGuide.rst

```diff
@@ -108,14 +116,13 @@ Syntax::
 
    <unique-id>-<test-name>[;<variant-id>]
 
-Examples of test-names::
-
-   '/bin/true'
-   'passtest.py:Passtest.test'
-   'file:///tmp/passtest.py:Passtest.test'
-   'multiple_tests.py:MultipleTests.test_hello'
-   'type_specific.io-github-autotest-qemu.systemtap_tracing.qemu.qemu_free'
+Example of Test IDs::
 
+   '001-/bin/true'
+   '002-passtest.py:Passtest.test;quiet-'
+   '003-file:///tmp/passtest.py:Passtest.test'
+   '004-multiple_tests.py:MultipleTests.test_hello;maximum_debug-df2f'
+   '005-type_specific.io-github-autotest-qemu.systemtap_tracing.qemu.qemu_free'
```

Comment:
ok, only point here is that `001` will only happen if the job has between 100 and 999 tests. For this example, I'd keep it coherent with the number of tests you're demonstrating...
```
'1-/bin/true'
'2-passtest.py:Passtest.test;quiet-'
...
```